applications/HDF5Application/python_scripts/hdf5_io.py

```diff
@@ -20,6 +20,8 @@ class IOObject(metaclass=ABCMeta):
     @abstractmethod
     def Execute(self, model_part, hdf5_file): pass
 
+    def ExecutePostOutput(self, model_part, hdf5_file_name): pass
```

Comment:
I added this bcs this way I can fully support the xdmf 
Plus I think this can also be useful for other outputs